Src/UUIDNext.Test/UUIDNext.Test.csproj

```diff
@@ -1,9 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net8.0</TargetFramework>
 
     <IsPackable>false</IsPackable>
+
+    <TargetFrameworks>net8.0;net472;net48</TargetFrameworks>
```

Comment:
net48 is the last netframework. Added it here just for tests to make sure all my assumptions about preprocessors works fine LOL